operations/helm/charts/mimir-distributed/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
-version: 3.0.0-rc.1
-appVersion: 2.1.0
+version: 3.0.0-rc.2
+appVersion: 2.2.0
 description: "Grafana Mimir"
 home: https://grafana.com/docs/mimir/v2.1.x/
```

Comment:
Should this be updated too?

Reply:
nice catch, we'll probably need some consistency check in CI